tests/test_dataset_registry.py

```diff
@@ -66,6 +66,9 @@
 
 @pytest.mark.parametrize("dataset_name", dataset_registry.keys())
 def test_dataset(dataset_name: str, dummy_dataloader_hparams: DataloaderHparams) -> None:
+    # Skip WebDatasets.
+    if dataset_name.startswith('wds_'):
```

Comment:
instead of by name, maybe check on whether the hparams is a subclass of the webdataset hparams?

Reply:
Done